site/content/docs/developing/svc-to-svc-communication.en.md

```diff
@@ -1,12 +1,48 @@
-# Service Discovery
+# Service-to-Service Communication
+
+## Service Connect
+
+!!! info
+    This feature is supported since Copilot `v1.24`, which provides more integrated service-to-service experience compared to [Service Discovery](#service-discovery).
+
+ECS Service Connect enables your client services connect to their dependencies in a load-balanced and resilient fashion, simplifying the way of exposing those dependencies to their clients. With Service Connect in Copilot, each service you create is given a private alias by default: `http://<your service name>`.
+
+### How do I use Service Connect?
+Imagine we have an app called `kudos` and two services: `api` and `front-end`, deployed in the same environment. In order to use Service Connect, both services' manifests need to have:
+
+```yaml
+network:
+  connect: true # Defaults to "false"
+```
+
+After deploying both services, they should be able to talk to each other using the default Service Connect endpoint, which is the same as its service name. For example, `front-end` service can simply call `http://api`.
+
+```go
+// Calling the "api" service from the "front-end" service.
+resp, err := http.Get("http://api/")
+```
```

Comment:
This reminds me (in a separate PR), I wonder if we should a recommended action after `copilot deploy` saying:
```
Your service is accessible within the VPC at: fmt.Sprintf(http://%s:%s/, name, port)
```

Reply:
I think I added that like

```
Recommended follow-up action:
  - You can access your service at redis-server:6379 with service connect.
```